t/web/spider_base.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../app'
+require 'pry'
```

Comment:
I don't think you want to leave that behind. (Though might be worth making a separate PR to just add it to `test_helper` so that all tests have it available)